scala/private/rules/scala_doc.bzl

```diff
@@ -43,6 +46,9 @@ def _scaladoc_aspect_impl(target, ctx):
 _scaladoc_aspect = aspect(
     implementation = _scaladoc_aspect_impl,
     attr_aspects = ["deps"],
+    attrs = {
+        "transitive": attr.string(default="true", values=["true", "false"]),
```

Comment:
Could this be boolean instead of string? ditto for the other.

Reply:
Setting this to `"transitive": attr.bool(default=True),` I get the following error,
```
ERROR: Traceback (most recent call last):
	File "/private/var/tmp/_bazel_rahuli/7534b642ae61a45da9baaf1cc06fb325/external/rules_scala/scala/private/rules/scala_doc.bzl", line 46, column 26, in <toplevel>
		_scaladoc_aspect = aspect(
Error in aspect: Aspect parameter attribute 'transitive' must have type 'string' and use the 'values' restriction.
```
So I believe we need to keep it a `string` with restricted `values`. I am no expert on `bazel` so happy to include a better way to do this

Reply:
According to [aspects doc](https://docs.bazel.build/versions/main/skylark/aspects.html) parameters must be of type string

> Public aspect attributes are of type string and are called parameters